flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/TableScanTest.scala

```diff
@@ -99,6 +99,25 @@ class TableScanTest extends TableTestBase {
     util.verifyExecPlan("SELECT * FROM MetadataTable")
   }
 
+  @Test
+  def testDDLWithMetadataThatConflictsWithPhysicalColumn(): Unit = {
```

Comment:
Should the same test be done for the sink table as well?

Reply:
+1